feature-flags/README.md

```diff
@@ -1,21 +1,27 @@
 # Feature Flags
 
 At Buoy we utilize feature flags to enable us to develop features without
-blocking merges into main or releasing often. To accomplish this we typically
-implement one or both of the following:
+blocking merges into main or releasing often.  Feature Flags can fall into one
+of 3 categories:
 
-- [LaunchDarkly] feature flags for client side control or for features which
-  might be per user dependent.
-- Server environment variables for toggling features on the server side.
+1. Release Toggle - This is a feature flag which is used to allow in progress
+   features to be merged into the main branch and go out to production "dark".
+2. Ops Toggles - A feature flag designed to allow us to incrementally release a
+   feature to production, typically targetted at the center level.
+3. Configuration Toggles - These are flags that enable / disable features on a
+   per customer or per environment basis and are not considered "temporary".
 
 ## Guide
 
+- For release toggles, prefer database backed feature flags
+- For configuration toggles, prefer environment variable backed feature flags
+- Avoid creating feature flags with LaunchDarkly. This is considered deprecated.
```

Comment:
There is still a "valid" case for `LaunchDarkly` feature flags: When there are two versions of the same page (React and Rails) and we want to be able to share it between client and server. Although we could come up with another approach if we don't want LD at all, like putting the feature flags in the browser's window so it can be picked up in the React context. Thoughts?

Reply:
You are right, that while we are still in React we may use LD. The intent behind this was that we shouldn't use LD unless it's something being used by React. I'll reword.